src/arduino/comms/packager.h

```diff
@@ -0,0 +1,20 @@
+#include <pressure.h>
+#include <potentiometer.h>
+#include <packet.h>
+// TODO: docstring
+class Packager
```

Comment:
This is likely all depreciated by new architecture

Reply:
Although some version of this should maybe exist for Future-proofing IMU stuff

Reply:
Removed references to potentiometers and pressure in the C++ Packager class. Now acts as more of a general data-transfer tool - once implemented it can be used for the pressure handshake and IMU stuff if we want to.